metrics/metrics.go

```diff
@@ -193,6 +195,12 @@ var (
 		ConstLabels: map[string]string{"build": common.COMMIT, "version": common.GetAppVersion().String()},
 	}, func() float64 { return float64(getBuildTimestamp(common.BUILDDATE)) })
 
+	// IsDrandNode is 1 for drand nodes, 0 for relays
+	IsDrandNode = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Why didn't you add a `metrics.IsDrandNode.Set(0)` to the relay code then?

Reply:
Because of the way the query language works, the metric only needs to be emitted on the drand node. It needs to have a value. Emitting it with a value of 0 in the other case actually makes the queries more complex. I've updated the description.